src/main/java/me/contaria/seedqueue/gui/wall/SeedQueueWallScreen.java

```diff
@@ -365,7 +365,9 @@ private void updateMainPreviews() {
                 break;
             }
             if (this.mainPreviews[i] == null && !this.blockedMainPositions.contains(i)) {
+                SeedQueuePreview preview = this.preparingPreviews.get(0);
                 this.mainPreviews[i] = this.preparingPreviews.remove(0);
+                preview.resetCooldown();
```

Comment:
Could you change this to `this.mainPreviews[i].resetCooldown()`?

Reply:
> Could you change this to `this.mainPreviews[i].resetCooldown()`?

done :D